docs/API/Release_Notes/API_Release_Notes.md

```diff
@@ -57,7 +57,8 @@
 
 ### Known Issues and Workarounds
 
-* Some fields may be repeated in the API or data portal due to being in a transitional states. As a workaround, select all of the fields of interest, and the results will demonstrate which are currently populated.
+* Survival plots are generated from the `diagnoses.days_to_last_follow_up` field. For some TCGA projects, in Data Release 42, data was migrated to the `follow_ups.days_to_follow_up` field. This resulted in an issue with missing cases for some TCGA projects in survival plots. The GDC is actively working on a fix. In the interim, users should create survival plots using the greatest value in the `follow_ups.days_to_follow_up` field.
+* Some fields may be repeated in the API or data portal due to being in a transitional states. As a workaround, select all of the fields of interest, and the results will demonstrate which are currently populated. <!-- SV-2584 -->
```

Comment:
small note that the SV-2584 comment goes with the survival plots line. (but users won't see this anyway.)

Reply:
Fixed!